src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs

```diff
@@ -83,7 +84,9 @@ public ActivityApi(PlatformEnvironment environment, bool continueOnCapturedConte
             catch (Exception e)
             {
                 Event(e, CreateLoggerArgs(EnumEventLevel.Error, $"GetActivitiesAsync Failed - {e.Message}"));
-                throw;
+                if (_throwAPIErrors) 
+					 throw; 
```

Comment:
I think that the spacing / tabs might be inconsistent. When I open a file in Visual Studio, I get the following message. Can you please fix this in the files you updated?

![image](https://user-images.githubusercontent.com/47400881/200026776-b15208f8-0af7-4a08-a18a-f06343a4182f.png)